test/junit/scala/collection/immutable/SeqMapTest.scala

```diff
@@ -39,4 +39,16 @@ class SeqMapTest {
     // `addAll`
     checkClass(build(_ ++= List(1 -> 1)), "SeqMap1")
   }
+
+  @Test def `keys are seqs`: Unit = {
+    val pairs = List.tabulate(6)(i => s"k$i" -> i)
+    for (elems <- pairs.tails) {
+      val vm = elems.to(VectorMap)
+      val sm = elems.to(SeqMap)
+      val keys = elems.map(_._1)
```

Comment:
I think we should test the type is a `Set` too.

Reply:
ListMap returns a list.